src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Input/Commands/WinFormsAddItemCommandHandler.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System.Collections.Generic;
+using System.ComponentModel.Composition;
+using Microsoft.VisualStudio.Input;
+using Microsoft.VisualStudio.Shell.Interop;
+
+namespace Microsoft.VisualStudio.ProjectSystem.VS.Input.Commands
+{
+    [ExportCommandGroup(CommandGroup.VisualStudioStandard2k)]
+    [AppliesTo(ProjectCapability.WindowsForms)]
+    internal class WinFormsAddItemCommandHandler : AbstractAddItemCommandHandler
+    {
+        // from VS: src\vsproject\cool\coolpkg\resource.h
+        protected enum LegacyCSharpStrings : uint
```

Comment:
You have two copies of this, can we just merge into a single enum instead of two enums with the same name?